stacks/lemp/service-config/nginx.conf

```diff
@@ -0,0 +1,82 @@
+worker_processes 4;
+pid /var/run/nginx.pid;
+
+events {
+	worker_connections 768;
+	# multi_accept on;
+}
+
+http {
+	# Basic Settings
+	sendfile on;
+	tcp_nopush on;
+	tcp_nodelay on;
+	keepalive_timeout 65;
+	types_hash_max_size 2048;
+	# server_names_hash_bucket_size 64;
+	server_tokens off;
+	server_name_in_redirect off;
+
+	include mime.types;
+	default_type application/octet-stream;
+
+	# Logging
+	access_log off;
+	error_log stderr;
+
+	# Prevent nginx from adding compression; this interacts badly with Sandstorm
+	# WebSession due to https://github.com/sandstorm-io/sandstorm/issues/289
+	gzip off;
+
+	# Trust the sandstorm-http-bridge's X-Forwarded-Proto.
+	map $http_x_forwarded_proto $fe_https {
+		default "";
+		https on;
+	}
+
```

Comment:
It sure would be nice if it were possible to share fractions of these config files between all stacks, but maybe `conf.d` etc is not worth it for app authors. So IMHO not a showstopper; I have no better solution; just wanted to remark on that.